targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Source/ExampleProject/PfTestActor.cpp.ejs

```diff
@@ -23,6 +23,13 @@ UPlayFabTestCase* APfTestActor::GetActiveTest()
     return activeTestCase;
 }
 
+UTestTitleDataLoader& APfTestActor::GetCachedTitleData()
+{
+    if (ttdLoader == nullptr)
+        ttdLoader = NewObject<UTestTitleDataLoader>();
+    return *ttdLoader;
+}
```

Comment:
Is there any risk of a race condition?  Does this ever run in a multithreaded environment?

Reply:
Since this is our own ExampleProject, and not part of the public SDK, I'm going to say: No.
Both paths for our ExampleProject are run on a single object on the global Unreal main thread, so there's no issue.